pkg/plugins/install.go

```diff
@@ -39,20 +45,37 @@ func (o *InstallOptions) Validate(args []string, cxt *portercontext.Context) err
 	return o.InstallOptions.Validate(args)
 }
 
-// InstallFileOption is the go representation of plugin installation file format.
-type InstallFileOption map[string]pkgmgmt.InstallOptions
+// InstallPluginsSpec represents the user-defined configuration for plugins installation.
+type InstallPluginsSpec struct {
+	SchemaType    string               `yaml:"schemaType"`
+	SchemaVersion string               `yaml:"schemaVersion"`
+	Plugins       InstallPluginsConfig `yaml:"plugins"`
+}
+
+func (spec InstallPluginsSpec) Validate() error {
+	if InstallPluginsSchemaVersion != schema.Version(spec.SchemaVersion) {
```

Comment:
I just remembered that we are missing schemaType validation on our other resources. I had created a [branch to fix](https://github.com/carolynvs/porter/tree/schema-type-validation) that and then got pulled off onto other things and forgot to submit a PR.

Let's do a check here for the schemaType as well, so that if they enter in a value other than what's expected, we don't try to process the file further. For example, they accidentally pass a bundle or installation file to the -f parameter.

https://github.com/carolynvs/porter/blob/ea694aa668a448eb2293775da5f5cfda3567214f/pkg/manifest/manifest.go#L185-L187

I'll submit my PR afterwards that adds the check to the other resources.